python/ccf/clients.py

```diff
@@ -35,7 +35,7 @@ def escape_loguru_tags(s):
     return loguru_tag_regex.sub(lambda match: f"\\{match[0]}", s)
 
 
-def truncate(string: str, max_len: int = 128):
+def truncate(string: str, max_len: int = 256):
```

Comment:
Since the member/user/node ids change, many responses were truncated too aggressively to display any useful information.